packages/rum_sdk/lib/src/rum_widgets_binding_observer.dart

```diff
@@ -2,20 +2,23 @@ import 'package:flutter/cupertino.dart';
 import 'package:rum_sdk/src/integrations/native_integration.dart';
 import 'package:rum_sdk/rum_sdk.dart';
 
-class RumWidgetsBindingObserver extends WidgetsBindingObserver{
+class RumWidgetsBindingObserver extends WidgetsBindingObserver {
+  AppLifecycleState? _previousState;
+
   @override
   void didChangeAppLifecycleState(AppLifecycleState state) {
     super.didChangeAppLifecycleState(state);
-    if(state==AppLifecycleState.resumed){
+    if (state == AppLifecycleState.resumed) {
       WidgetsBinding.instance!.addPostFrameCallback((timeStamp) {
         NativeIntegration.instance.getWarmStart();
       });
       NativeIntegration.instance.setWarmStart();
     }
-  }
-  @override
-  void didHaveMemoryPressure() {
-    super.didHaveMemoryPressure();
-  }
 
-}
\ No newline at end of file
+    RumFlutter().pushEvent("app_lifecycle_changed", attributes: {
```

Comment:
We had a use case where it would be interesting to see lifecycle changes. That is why I added this.